recipes/kompose/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set organization = "kubernetes" %}
+{% set repository = "kompose" %}
+{% set version = "1.28.0" %}
+
+package:
+  name: {{ repository }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ organization }}/{{ repository }}.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0 
+  script:
+    - go build -o %LIBRARY_BIN%{{ os.sep }}{{ repository }}.exe main.go             # [win]
+    - go build -o $BUILD_PREFIX{{ os.sep }}bin{{ os.sep }}{{ repository }} main.go  # [unix]  
+
+requirements:
+  build:
+    - go
+
+test:
+  commands:
+    - {{ repository }} --help
+
+about:
+  home: https://github.com/{{ organization }}/{{ repository }}
+  summary: 'Kompose is a conversion tool for Docker Compose to container orchestrators such as Kubernetes (or OpenShift).'
+  description: |
+    Kompose is a tool to help users who are familiar with 
+    docker-compose <https://docs.docker.com/compose/> move 
+    to Kubernetes <http://{{ organization }}.io>. It takes a 
+    Docker Compose file and translates it into Kubernetes resources.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  doc_url: https://{{ repository }}.io/
+  dev_url: https://github.com/{{ organization }}/{{ repository }}
```

Comment:
I would prefer if you use less templating in this section. It makes it much more readable.